x-pack/test_serverless/functional/test_suites/security/ftr/cases/configure.ts

```diff
@@ -26,7 +26,7 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
 
   describe('Configure Case', function () {
     before(async () => {
-      await svlCommonPage.login();
+      await svlCommonPage.loginAsAdmin();
```

Comment:
Why is this one `loginAsAdmin` vs `loginWithPrivilegedRole` in the other ones?
(I'm checking the cases tests)

Reply:
@adcoelho it is actually a good question: the test should be using the least required privileges and I believe that Tre' tried it first with Editor  (`loginWithPrivilegedRole` is using Editor/Developer based on project type) and it failed. Then we run the test with highest access role - `Admin`.
If you believe that cloud user with Editor role still should have access it, please open an issue and investigate it. Appex-QA relies on code owners to validate the selected roles.

Reply:
@adcoelho may I have one more review please with coffee and donuts?! 😄